litmus-portal/frontend/src/pages/ApplicationDashboard/styles.ts

```diff
@@ -96,6 +96,21 @@ const useStyles = makeStyles((theme) => ({
     background: theme.palette.background.paper,
     borderRadius: 4,
     textTransform: 'none',
+    '&:hover': {
+      borderColor: theme.palette.primary.main,
+      boxShadow: `0px 4px 5px -2px ${fade(
+        theme.palette.highlight,
+        0.2
+      )},0px 7px 10px 1px ${fade(
+        theme.palette.highlight,
+        0.14
+      )},0px 2px 16px 1px ${fade(theme.palette.highlight, 0.12)}`,
+      background: theme.palette.background.paper,
+    },
+    '&$focused': {
+      borderColor: theme.palette.primary.main,
+      background: theme.palette.background.paper,
+    },
```

Comment:
Can property background removed from focused as its same as parent

Reply:
On removing it the style is getting overridden explicitly to card header colour and then the background and drop down focused background become of same colour but adding this explicitly solved the issue @whitetiger1399 we need the explicit override here I have added at both places where error was happening

Reply:
Ok